src/molecule/util.py

```diff
@@ -224,29 +255,50 @@ def safe_dump(data: Any, explicit_start=True) -> str:
     )
 
 
-def safe_load(string) -> Dict:
+def new_safe_loader_with_include(import_path: str):
```

Comment:
When is "new"? Today yes but tomorrow not. I would just go with `safe_loader_with_include`

Reply:
True :-) Fixed